Dockerfile

```diff
@@ -2,16 +2,17 @@ FROM openjdk:8u212-jre
 
 WORKDIR /opt
 
-ENV HADOOP_HOME=/opt/hadoop-2.9.2
-ENV HIVE_HOME=/opt/apache-hive-2.3.5-bin
+ENV HADOOP_HOME=/opt/hadoop-3.1.3
+ENV HIVE_HOME=/opt/apache-hive-metastore-3.0.0-bin
 # Include additional jars
-ENV HADOOP_CLASSPATH=/opt/hadoop-2.9.2/share/hadoop/tools/lib/aws-java-sdk-bundle-1.11.199.jar:/opt/hadoop-2.9.2/share/hadoop/tools/lib/hadoop-aws-2.9.2.jar
+ENV HADOOP_CLASSPATH=/opt/hadoop-3.1.3/share/hadoop/tools/lib/aws-java-sdk-bundle-1.11.271.jar:/opt/hadoop-3.1.3/share/hadoop/tools/lib/hadoop-aws-3.1.3.jar
 
-RUN apt-get update && \
-    curl -L https://www-us.apache.org/dist/hive/hive-2.3.5/apache-hive-2.3.5-bin.tar.gz | tar zxf - && \
-    curl -L https://www-us.apache.org/dist/hadoop/common/hadoop-2.9.2/hadoop-2.9.2.tar.gz | tar zxf -
-
-COPY conf ${HIVE_HOME}/conf
+RUN curl -L https://www-us.apache.org/dist/hive/hive-standalone-metastore-3.0.0/hive-standalone-metastore-3.0.0-bin.tar.gz | tar zxf - && \
+    curl -L https://www-us.apache.org/dist/hadoop/common/hadoop-3.1.3/hadoop-3.1.3.tar.gz | tar zxf - && \
+    rm -f ${HIVE_HOME}/lib/guava-19.0.jar && \
```

Comment:
Could you explain why you have this removal and then replacement of a specific jar?

Reply:
> The trick of altering `guava-19.0.jar` with `guava-27.0-jre.jar` from Hadoop package comes from https://blog.csdn.net/weixin_43231331/article/details/103066837